HW4_Sapozhnikov/prototool.py

```diff
@@ -0,0 +1,564 @@
+            'MOLECULAR_WEIGHTS': 133},
+    'Cys': {'TO_1': 'C',
+            'PROTEIN_TO_RNA_COMBINATION': {'UGU', 'UGC'},
+            'PKA_AMINOACIDS': [1.96, 10.28, 8.18],
+            'MOLECULAR_WEIGHTS': 121},
+    'Glu': {'TO_1': 'Q',
+            'PROTEIN_TO_RNA_COMBINATION': {'GAA', 'GAG'},
+            'PKA_AMINOACIDS': [2.19, 9.67, 4.25],
+            'MOLECULAR_WEIGHTS': 147},
+    'Gln': {'TO_1': 'E',
+            'PROTEIN_TO_RNA_COMBINATION': {'CAA', 'CAG'},
+            'PKA_AMINOACIDS': [2.17, 9.13],
+            'MOLECULAR_WEIGHTS': 146},
+    'Gly': {'TO_1': 'G',
+            'PROTEIN_TO_RNA_COMBINATION': {'GGU', 'GGC', 'GGA', 'GGG'},
+            'PKA_AMINOACIDS': [2.34, 9.60],
+            'MOLECULAR_WEIGHTS': 75},
+    'His': {'TO_1': 'E',
+            'PROTEIN_TO_RNA_COMBINATION': {'CAU', 'CAC'},
+            'PKA_AMINOACIDS': [1.82, 9.17],
+            'MOLECULAR_WEIGHTS': 155},
+    'Ile': {'TO_1': 'I',
+            'PROTEIN_TO_RNA_COMBINATION': {'AUU', 'AUC', 'AUA'},
+            'PKA_AMINOACIDS': [2.36, 9.68],
+            'MOLECULAR_WEIGHTS': 131},
+    'Leu': {'TO_1': 'L',
+            'PROTEIN_TO_RNA_COMBINATION': {'CUU', 'CUC', 'CUA', 'CUG'},
+            'PKA_AMINOACIDS': [2.36, 9.60],
+            'MOLECULAR_WEIGHTS': 131},
+    'Lys': {'TO_1': 'K',
+            'PROTEIN_TO_RNA_COMBINATION': {'AAA', 'AAG'},
+            'PKA_AMINOACIDS': [2.18, 8.95, 10.53],
+            'MOLECULAR_WEIGHTS': 146},
+    'Met': {'TO_1': 'M',
+            'PROTEIN_TO_RNA_COMBINATION': {'AUG'},
+            'PKA_AMINOACIDS': [2.28, 9.21],
+            'MOLECULAR_WEIGHTS': 149},
+    'Phe': {'TO_1': 'F',
+            'PROTEIN_TO_RNA_COMBINATION': {'UUU', 'UUC'},
+            'PKA_AMINOACIDS': [2.20, 9.13],
+            'MOLECULAR_WEIGHTS': 165},
+    'Pro': {'TO_1': 'P',
+            'PROTEIN_TO_RNA_COMBINATION': {'CCU', 'CCC', 'CCA', 'CCG'},
+            'PKA_AMINOACIDS': [1.99, 10.96],
+            'MOLECULAR_WEIGHTS': 115},
+    'Ser': {'TO_1': 'S',
+            'PROTEIN_TO_RNA_COMBINATION': {'UCU', 'UCC', 'UCA', 'UCG'},
+            'PKA_AMINOACIDS': [2.21, 9.15],
+            'MOLECULAR_WEIGHTS': 105},
+    'Thr': {'TO_1': 'T',
+            'PROTEIN_TO_RNA_COMBINATION': {'ACU', 'ACC', 'ACA', 'ACG'},
+            'PKA_AMINOACIDS': [2.11, 9.62],
+            'MOLECULAR_WEIGHTS': 119},
+    'Tyr': {'TO_1': 'W',
+            'PROTEIN_TO_RNA_COMBINATION': {'UAU', 'UAC'},
+            'PKA_AMINOACIDS': [2.20, 9.11, 10.07],
+            'MOLECULAR_WEIGHTS': 181},
+    'Trp': {'TO_1': 'Y',
+            'PROTEIN_TO_RNA_COMBINATION': {'UGG'},
+            'PKA_AMINOACIDS': [2.38, 9.39],
+            'MOLECULAR_WEIGHTS': 204},
+    'Val': {'TO_1': 'V',
+            'PROTEIN_TO_RNA_COMBINATION': {'GUU', 'GUC', 'GUA', 'GUG'},
+            'PKA_AMINOACIDS': [2.32, 9.62],
+            'MOLECULAR_WEIGHTS': 117},
+}
+
+# A dictionary where keys are 1-letter and values are 3-letters codes
+TO_3_DICT = {nested_dict['TO_1']: key for key,
```

Comment:
Прикольно что не хардкодили все! Потому что если нужно будет изменить что-то - так можно это сделать всего один раз в главном словаре, а тут все уже будет автоматически. Разве что имя `nested_dict` я не сразу осознал что значит. Мб можно что-то получше придумать, но не знаю.
```suggestion
TO_3_DICT = {nested_dict['TO_1']: key for key,
```